elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -68,7 +68,7 @@ function update_indent(leftDiff, id, ui) {
 
     // leftDiff is the direction to move the MCQ answer tile, in px
     // we limit leftDiff to be increments of TABWIDTH, whether positive or negative
-    if (currentIndent != ''){
+    if (currentIndent !== ''){
```

Comment:
I think using `!==` would work, the intention is to check whether a block is indented by checking the `marginLeft` CSS style is set, as in `var currentIndent = ui.item[0].style.marginLeft;` 10 lines earlier. 

I can't immediately say whether checking `if (currentIndent)` would work, because I don't know too much about Javascript truthy falsy to say if changing the check to `if (currentIndent)` would add any unintentional behaviour (because javascript). Would the javascript wizards of this repo mind chiming in? There might be another much better way to check if the CSS property is set tho.